### DISCUSSION TITLE
[feature suggest] active learning in lammps

### DISCUSSION THREAD

**Discussion by thangckt** [ACEsuit/mace#574] [Ideas]
Dear Developers,

I wonder if you have any plan to implement an uncertainty quantification to use for active learning in Lammps?
I saw the example [active_learning_md.py](https://github.com/ACEsuit/mace/blob/develop/mace/cli/active_learning_md.py) use ASE as exploration engine.

Can you have a little guide on how to get such thing with LAMMPS?

Thank you so much

**Comment by ilyes319** [comment]
There is currently no plan to support that as the MACE level. I recommend you to either use the ASE interface, or try to find a way to create a custum LAMMPS script that takes a committee. I will turn that into a discussion to let other contribute if they have ideas.
